### ISSUE TITLE
[BWL] Infight Bug nach Razorgore

### ISSUE THREAD

**Issue by Nifexx** [ClassicWoW/Nefarian_1.12.1_Bugtracker#2354]
Welches Verhalten wird beobachtet?
Die letzten Male gab es einen Infight Bug, nachdem die Techniker (welche ein super merkwürdiges Verhalten an den Tag legen) getötet wurden.

Wie sollte es sich verhalten?
Kein Infight Bug

Schritte zur Reproduzierung
Razorgore töten.

Zusätzliche Informationen (Screenshots, Videos, Klasse, Rasse, Level, etc.)
-
Alle Kreaturen, Items, Objekte, Quests, Zauber etc. müssen aus unserer Datenbank verlinkt sein.
-

**Comment by Sheed90** [comment]
Können wir bestätigen, am besten einem GM im Discord Bescheid geben, wenn ein Raid stattfindet. Dieser kann ja gucken, mit welchem Mob man infight ist.

Gruß

**Comment by Dotox** [comment]
Am einfachsten wäre es, wie Sheed bereits gesagt hat, bei einem GM bzw. am Besten bei mir melden.

**Comment by Sheed90** [comment]
Wir gehen am Donnerstag um 20 Uhr wieder BWL. Ich schreibe dich dann an. :-)

**Comment by Sheed90** [comment]
Hallo, auch wenn wir uns gestern nicht "treffen" konnten, so konnten wir feststellen, dass auf jeden Fall die Techniker Schuld sind. Daher:

Schritte zur Reproduzierung
Razorgore töten, prüfen ob man infight bleibt.
Techniker bei Vaelastrasz töten, erneut prüfen, ob man infight bleibt (vermutlich ja).
Anschließend prüfen, mit welchem Monster man infight ist. 

Gruß

**Comment by Herlot** [comment]
Kann ebenfalls bestätigen, dass es sehr sicher an den Technikern liegt. Wir beobachten das Verhalten auch jede Woche.

**Comment by Ulva-Anorhin** [comment]
Push! 
Jedesmal nach der 1. Techniker Gruppe vor vaelestrah

**Comment by cyberenix** [comment]
Meine Vermutung dazu ist, dass jedes Mal wenn die Techniker gepullt werden, Vael mehr oder weniger mitgeaddet wird. Man hört jedes Mal ein Brüllen von ihm sobald die Techniker gepullt werden.

**Comment by Sheed90** [comment]
Naja das Problem wäre relativ einfach zu identifizieren. Ein GM hat die Möglichkeit zu prüfen, mit welchen NPCs der Spielet im Kampf ist. Dago und Dotox haben diesen Befehl extra eingefügt. 

Nur müsste sich eine Gruppe mal Zeit nehmen und warten bis der GM vor Ort ist.

**Comment by dagochen** [comment]
Tritt das überhaupt noch auf?

**Comment by Sheed90** [comment]
Ja, nur haben die meisten Gruppen ihre eigene Lösung dafür gefunden. Sprich: der Raid verlässt geschlossen die Instanz und betritt sie danach wieder = outfight  :P

**Comment by dagochen** [comment]
Einfach direkt mich im Discord anschreiben und highlighten, wenn das Problem auftritt. Ich kann dann nachschauen, welche Kreaturen mit den Spielern infight sind. Nur so können wir das Problem beheben!

**Comment by Sheed90** [comment]
Update: dago hat uns gestern direkt begleitet und das Problem überprüft. Leider konnte keine Quelle gefunden werden, die zeigt, warum ca. 1/3 des Raids infight sind. 

Ich stand als PM noch an Razergore während vorne gepullt wurde. Auch ich kam infight, obwohl ich mit dem Kampf an sich gar nichts zu tun hatte.
